Flag --show-all has been deprecated

DaveVoyles opened the pull request:
```$ kubectl get pods --selector app=samples-tf-mnist-demo --show-all```
When running this command, the CLI returns this warning:

```Flag --show-all has been deprecated, will be removed in an upcoming release```


If you run the command **without** --show-all, it still returns the nodes as expected:
```
 DaveVoyles@minint-55r162g  ~/Desktop  kubectl get pods --selector app=samples-tf-mnist-demo --show-all
Flag --show-all has been deprecated, will be removed in an upcoming release
NAME                          READY     STATUS      RESTARTS   AGE
samples-tf-mnist-demo-jhsgf   0/1       Completed   0          28m
 DaveVoyles@minint-55r162g ~/Desktop  kubectl get pods --selector app=samples-tf-mnist-demo
NAME                          READY     STATUS      RESTARTS   AGE
samples-tf-mnist-demo-jhsgf   0/1       Completed   0          29m```

azurecla commented:
Hi __@rdtechie__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@rdtechie : Thanks for your contribution to the Azure documentation! The author, @coreyp-at-msft, has been notified to review your proposed change.

coreyp-at-msft commented:
#sign-off

DaveVoyles committed:
Flag --show-all has been deprecated

```$ kubectl get pods --selector app=samples-tf-mnist-demo --show-all```
When running this command, the CLI returns this warning:

```Flag --show-all has been deprecated, will be removed in an upcoming release```


If you run the command **without** --show-all, it still returns the nodes as expected:
```
 DaveVoyles@minint-55r162g  ~/Desktop  kubectl get pods --selector app=samples-tf-mnist-demo --show-all
Flag --show-all has been deprecated, will be removed in an upcoming release
NAME                          READY     STATUS      RESTARTS   AGE
samples-tf-mnist-demo-jhsgf   0/1       Completed   0          28m
 DaveVoyles@minint-55r162g ~/Desktop  kubectl get pods --selector app=samples-tf-mnist-demo
NAME                          READY     STATUS      RESTARTS   AGE
samples-tf-mnist-demo-jhsgf   0/1       Completed   0          29m```
